pkg/boot/fit/vfit.go

```diff
@@ -0,0 +1,211 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// Performs signature checks on FDT images.
+// Currently supports PGP and raw PKCS1v15 RSA signatures.
+//
+// Expected FDT Format:
+//  Node: images
+//  Node: image_name
+//   P: data
+//   Node: signature*
+//    P: value
+//    P: algo          (ex. 'sha256,rsa4096', 'pgp')
```

Comment:
Can you clarify that the hash algorithm must come before the signing algorithm here? (Or point to a more detailed documentation of the format?)

Reply:
Fixed it to return the first in a scan. Updated comment.
Squashed